api/v1alpha1/httpchaos_types.go

```diff
@@ -20,6 +20,8 @@ import (
 )
 
 // +kubebuilder:object:root=true
+// +kubebuilder:printcolumn:name="action",type=string,JSONPath=`.spec.action`
+// +kubebuilder:printcolumn:name="duration",type=string,JSONPath=`.spec.duration`
```

Comment:
`HTTPChaos` does not contains a field `.spec.action`

Reply:
So I need to remove this field or change the XXXXChaosSpec in that situation?

Reply:
Please remove this comment `+kubebuilder:printcolumn:name="action"` with this situation :)